packages/lib/server/queries/booking/index.ts

```diff
@@ -10,14 +10,15 @@ export const getTotalBookingDuration = async ({
   endDate: Date;
 }) => {
   // Aggregates the total booking time for a given event in a given time period
-  const [totalBookingTime] = (await prisma.$queryRaw`
+  // FIXME: bookings that overlap on one side will never be counted
+  const [totalBookingTime] = await prisma.$queryRaw<[{ totalMinutes: number | null }]>`
     SELECT SUM(EXTRACT(EPOCH FROM ("endTime" - "startTime")) / 60) as "totalMinutes"
     FROM "Booking"
     WHERE "status" = 'accepted'
-      AND "id" = ${eventId}
+      AND "eventTypeId" = ${eventId}
```

Comment:
mixup between booking and event id, caused booking limits to not be enforced here and lead to unnecessary calls to getUserAvailability where those limits were checked again